src/praisonai/praisonai/adapters/vector_stores.py

```
@@ -23,6 +23,20 @@ class ChromaVectorStore:
     """ChromaDB vector store adapter."""
     
     name: str = "chroma"
+    _COLLECTION_METADATA = {"hnsw:space": "cosine"}
```

Ruff (0.16.0)

[warning] Mutable default value for class attribute

(RUF012)